source/eventHandler.py

```diff
@@ -499,11 +499,18 @@ def shouldAcceptEvent(eventName, windowHandle=None):
 
 	fg = winUser.getForegroundWindow()
 	fgClassName=winUser.getClassName(fg)
-	if wClass == "NetUIHWND" and fgClassName in ("Net UI Tool Window Layered","Net UI Tool Window"):
+	if (
 		# #5504: In Office >= 2013 with the ribbon showing only tabs,
 		# when a tab is expanded, the window we get from the focus object is incorrect.
 		# This window isn't beneath the foreground window,
-		# so our foreground application checks fail.

```

Comment:
I think this should be kept, or the previous line should get the comma replaced with a period

Reply:
Note that I kept that line, but moved it further below the if check.